src/adapters/postgres/user-adapter.ts

```diff
@@ -255,10 +255,33 @@ export class PostgresUserService {
       delete whereClause.userId;
       whereClause.username = username;
     }
-    let userDetails = await this.usersRepository.findOne({
+    let getUserDetails = await this.usersRepository.findOne({
```

Comment:
variable should be `userDetails` only

Reply:
Also, you can pass the `select` json to the findone, to get the selected fields.
check here
https://stackoverflow.com/questions/59645009/how-to-return-only-some-columns-of-a-relations-with-typeorm

Reply:
Also, you can pass the select json to the findone, to get the selected fields.
check here
https://stackoverflow.com/questions/59645009/how-to-return-only-some-columns-of-a-relations-with-typeorm
@souravbhowmik1999